public/js/components/subscription.jsx

```diff
@@ -43,12 +43,12 @@ export default class Subscription extends Component {
         <div className="actions">
           <label className="vh"
             htmlFor="pm">{gettext('Change payment account')}</label>
+          {this.props.payMethods.length ?
           <PayMethodDropDown
```

Comment:
this feels like it should be indented to indicate that the logical block is still in progress